nuxt.config.ts

```diff
@@ -99,7 +99,10 @@ export default defineNuxtConfig({
     stripePublicKey: '',
     stripeWebhookSecret: '',
     public: {
-      appUrl: '',
+      appUrl: process.env.NUXT_PUBLIC_APP_URL || '',
```

Comment:
`process.env` won't work, please read https://nuxt.com/docs/guide/going-further/runtime-config and watch a video from Alexander Lichter showcasing the top mistake developers make using runtimeConfig.